docs/sources/mimir/configure/configure-zone-aware-replication.md

```diff
@@ -87,6 +89,16 @@ Deploying Grafana Mimir with zone-aware replication across multiple cloud provid
 
 The [Kubernetes Rollout Operator](https://github.com/grafana/rollout-operator) is a Kubernetes operator that makes it easier for you to manage multi-availability-zone rollouts. Consider using the Kubernetes Rollout Operator when you run Grafana Mimir on Kubernetes with zone awareness enabled.
 
+Note that even in a scenario where you have deployed Grafana Mimir to a single physical availability zone, the Rollout Operator is immensely useful.
```

Comment:
^ Could you please capitalize Operator in line 90 to follow the standard we've set for capitalizing K8 Operators to distinguish from human operators.